setup.py

```diff
@@ -76,7 +76,7 @@ def get_git_sha():
         "croniter>=0.3.28",
         "cryptography>=2.4.2",
         "flask>=1.1.0, <2.0.0",
-        "flask-appbuilder>=2.3.1, <2.4.0",
+        "flask-appbuilder==2.3.2rc1",
```

Comment:
nice, this makes more sense

Reply:
I actually want to revert this before changing and after flask-appbuolder 2.3.2 is out